py/shared/utils.py

```diff
@@ -30,11 +30,12 @@ class Currencies(str, Enum):
 }
 
 
-def slugify(title):
+def slugify(title, max_length: int = 255):
```

Comment:
```suggestion
def slugify(title, max_length: Optional[int] = None):
```
and only truncate when `max_length is not None`, also you should use [`shorten`](https://docs.python.org/3/library/textwrap.html) to cut at a word boundary.

Reply:
Good idea. Done